test-graalvm/common/src/main/java/testgraalvm/controllers/dto/OwnerDto.java

```diff
@@ -15,15 +15,24 @@
  */
 package testgraalvm.controllers.dto;
 
-import io.micronaut.core.annotation.Introspected;
+import io.micronaut.serde.annotation.Serdeable;
 
-@Introspected
+@Serdeable
 public class OwnerDto {
 
     private Long id;
     private String name;
     private int age;
 
+    public OwnerDto() {
+    }
+
+    public OwnerDto(Long id, String name, int age) {
```

Comment:
should this constructor be annotated with `@Creator`?

Reply:
Not needed